src/vmm/src/builder.rs

```diff
@@ -622,12 +589,20 @@ fn create_vcpus_aarch64(
 /// Configures the system for booting Linux.
 pub fn configure_system_for_boot(
     vmm: &Vmm,
-    vcpus: &[Vcpu],
+    vcpus: &mut [Vcpu],
+    _vcpu_config: VcpuConfig,
```

Comment:
Is there a better way to make this parameter x86_64 specific and not work around compiler unused warning ?